adc/ad7779/ad7779.c

```diff
@@ -543,25 +564,35 @@ static int ad7779_reset(int hard)
 	}
 
 	/* Software reset */
-	ad7779_gpio(start, 0);
-	usleep(10000);
 	ad7779_gpio(reset, 0);
-	usleep(200000);
-	ad7779_gpio(start, 1);
-	usleep(100000);
+	usleep(2);
 	ad7779_gpio(reset, 1);
+	usleep(300);
 
 	memset(status, 0, sizeof(status));
 
 	for (i = 0; i < 4; ++i) {
-		if (!ad7779_get_status(status)) {
-			if (status[16] & 0x10)
-				return AD7779_OK;
+		if (ad7779_get_status(status)) {
+			continue;
+		}
```

Comment:
Logic have changed, maybe usleep(100000) from line 592  should be moved above continue, like:
```suggestion
		if (ad7779_get_status(status) != AD7779_OK) {
			usleep(100 * 1000);
			continue;
		}
```

Reply:
Thanks! I will fix it